File: 2/02954.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
	string sentence;
	getline(cin, sentence);

	string result = "";
	for (int i = 0; i < sentence.length(); i++) {
		if (sentence[i] == 'a' || sentence[i] == 'e' || sentence[i] == 'i'
			|| sentence[i] == 'o' || sentence[i] == 'u') {
			result += sentence[i];
			i += 2;
		}
		else result += sentence[i];
	}

	cout << result << endl;
}

File: 2/02954.py
sentence = input()

result = ""
no_copy = False
for i in range(len(sentence)):
    if no_copy:
        if sentence[i] == 'p': continue
        else: no_copy = False
    else:
        if sentence[i] == 'a' or sentence[i] == 'e' or sentence[i] == 'i' \
            or sentence[i] == 'o' or sentence[i] == 'u':
            result += sentence[i]
            no_copy = True
        else: result += sentence[i]

print(result)
